src/reducers/petitions.js

```diff
@@ -20,6 +22,14 @@ export default function petitions (state = initialState, action) {
           qs: action.qs || ''
         }
       );
+    case GET_CITY:
```

Comment:
Genuine question - why do we need two `city` state items?
Isn't the `city` already defined in `cityFilterValue.data` ?

Or have I missed something?